### PR TITLE
Update docs to include examples

### PR DESCRIPTION
**Pull request by Racer159**
Update the docs site to include the updated examples.  (Note `Edit this page` currently references `master` so some pages will not exist until merging https://github.com/defenseunicorns/zarf/pull/681)

Fixes: #7 #46 #50

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,4 @@
-[submodule "zarf-repo"]
-	path = zarf-repo
+[submodule "master"]
+	path = master
 	url = https://github.com/defenseunicorns/zarf.git
+    branch = 676-example-readmes-are-out-of-date
```
Comment (Racer159) [inline .gitmodules:4]: The branch will need to be dropped before merge!

(also `zarf-repo` was changed to `master` to fix a bug with the Edit URL not working correctly)

Comment (Racer159) [reply .gitmodules:4]: Note also that we are pulling in all .md files now which has the following side effects:

1. Building the site takes a little bit longer (Con)
2. Docusaurus will error out on any bad link or markdown syntax error to tell us to update them (Pro)
3. We can import documents from anywhere in the repo (like the CONTRIBUTING.md file) (Pro)

**Comment by Racer159** [comment]
Note that we are pulling in all .md files now which has the following side effects:

    Building the site takes a little bit longer (Con)
    Docusaurus will error out on any bad link or markdown syntax error to tell us to update them (Pro)
    We can import documents from anywhere in the repo (like the CONTRIBUTING.md file) (Pro)

**Comment by Madeline-UX** [comment]
@Racer159 I have some feedback to add. Can we tag up to discuss a few design things - information architecture & admonition class?

**Comment by Racer159** [comment]
> 

Sure! Just let me know when you are available.

**Comment by Madeline-UX** [comment]
## Design Feedback ##

- [ ] Change Admonation class on index page from info to caution - since this information is important and should not be overlooked. 
- [ ] Information architecture for Exaples (including package examples and walkthroughs under the same category)

**Review by Madeline-UX — APPROVED** [review]

**Pull request merged**